File: app/src/main/java/com/imangazalievm/bubbble/di/shotslist/ShotsPresenterModule.java
package com.imangazalievm.bubbble.di.shotslist;

import com.imangazalievm.bubbble.di.global.scopes.Presenter;

import dagger.Module;
import dagger.Provides;

@Module
public class ShotsPresenterModule {

    private final String sortType;

    public ShotsPresenterModule(String sortType) {
        this.sortType = sortType;
    }

    @Provides
    @Presenter
    String provideSortType() {
        return sortType;
    }

}

File: app/src/main/java/com/imangazalievm/bubbble/di/shotslist/ShotsPresenterModule.kt
package com.imangazalievm.bubbble.di.shotslist

import com.imangazalievm.bubbble.di.global.scopes.Presenter
import dagger.Module
import dagger.Provides

@Module
class ShotsPresenterModule(private val sortType: String) {

    @Provides
    @Presenter
    fun provideSortType(): String = sortType

}